.github/workflows/health-metrics-presubmit.yml

```diff
@@ -3,7 +3,7 @@ name: Health Metrics
 on: [push, pull_request]
 
 env:
-  METRICS_SERVICE_URL: ${{ secrets.METRICS_SERVICE_URL }}
+  METRICS_SERVICE_URL: "https://api.firebase-sdk-health-metrics.com"
```

Comment:
I think you removed it in the other PR?

Reply:
Yes. Once #5706  is merged, I'll don't know whether I'll need to address the merge conflict here or GitHub is smart enough to handle it. Let's see.